templates/crossplane/pkg/controller.go.tpl

```diff
@@ -180,22 +180,30 @@ func newExternal(kube client.Client, client svcsdkapi.{{ .SDKAPIInterfaceTypeNam
 	e := &external{
 		kube:           kube,
 		client:         client,
-		{{- if or .CRD.Ops.ReadOne .CRD.Ops.GetAttributes .CRD.Ops.ReadMany }}
+		{{- if or .CRD.Ops.ReadOne }}
```

Comment:
I think this needs to be either this:

```
{{- if or .CRD.Ops.ReadOne .CRD.Ops.GetAttributes }}
```

or

```
{{- if .CRD.Ops.ReadOne }}
```

Reply:
Good catch, fixed! Thanks!